scripts/supercloud/run_pg3_tamp_experiments.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# Note: this script is too large to be run all at once. Comment things out
+# and run in multiple passes. For example, start with just 1000 train tasks.
+# If that looks good, launch the environments in separate runs.
+
+FILE="scripts/supercloud/submit_supercloud_job.py"
+# Note: this script is meant to be run first, to completion, with
+# RUN_LOAD_EXPERIMENTS=false, then rerun with RUN_LOAD_EXPERIMENTS=true.
+RUN_LOAD_EXPERIMENTS=false
+ALL_NUM_TRAIN_TASKS=(
```

Comment:
let's actually make the default here a few different values. maybe 10, 50, and 100? and leave them all uncommented.